Cartfile.resolved

```diff
@@ -1,5 +1,5 @@
 binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" "4.5.0"
-binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" "3.1.5"
+binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" "3.2.0"
```

Comment:
I think you'll get a failure of you bump the MapboxCoreNavigation.podspec to 3.2.0. Don't forget to run `pod update --repo update` from `MapboxCoreNavigationTests/CocoaPodsTest/PodInstall` after you've changed the podspec.

Reply:
@frederoni changes made in this [commit](https://github.com/mapbox/mapbox-navigation-ios/pull/1792/commits/bb77023270a4a7894e7d9704156fd38a1ff5feef)